config/kinoite-packages.yml

```diff
@@ -19,6 +23,7 @@ remove:
   - flatpak-selinux
   - flatpak-kcm
   - flatpak-session-helper
+  - plasma-discover-flatpak
```

Comment:
please remove any changes where you moved existing lines around

Reply:
they where sorted nonalphabetically, so I thought to put them at the top